### ISSUE TITLE
Wrong link to latest docs in old versions

### ISSUE THREAD

**Issue by jeikabu** [nanomsg/nng#1128]
The top of [1.0.0](https://nng.nanomsg.org/man/v1.0.0/index.html) pages says:
```
This documentation is for version 1.0.0 of nng, but the latest released version is 1.2.2. Please see the documentation for 1.2.2 for the most up-to-date information.
```
With a link that goes to [https://nng.nanomsg.org/man/v1.2.2/nng.html](https://nng.nanomsg.org/man/v1.2.2/nng.html) while it should be [https://nng.nanomsg.org/man/v1.2.2/index.html](https://nng.nanomsg.org/man/v1.2.2/index.html)

**Comment by gdamore** [comment]
Yeah, I have to go back and regenerate the static content for the new link locations.  What a pain.

**Comment by gdamore** [comment]
This should all be fixed now.  I've also published the v1.2.4 docs (which are the same), and we are no longer embedding the CSS which saves a *lot* on the download (you can cache).
